modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3871,6 +3871,51 @@
 		"issueKey": "LPD-50175",
 		"to": "fetchAccountEntryByExternalReferenceCode(param#1#, param#0#)"
 	},
+	{
+		"classNames": [
+			"CommercePriceEntryLocalService",
+			"CommercePriceEntryLocalServiceUtil"
+		],
+		"from": "addCommercePriceEntry(long paramName, long paramName, BigDecimal paramName, BigDecimal paramName, ServiceContext paramName)",
+		"issueKey": "LPD-51505",
```

Comment:
@regiCesar21 should be the task LPD not the subtask. The sub-task doesn't contains the pattern explanation